cql3/CqlParser.g4

```diff
@@ -637,11 +644,11 @@ columnList
    ;
 
 expressionList
-   : (constant | assignmentMap | assignmentSet | assignmentList) (syntaxComma (constant | assignmentMap | assignmentSet | assignmentList))*
+   : (constant | assignmentMap | assignmentSet | assignmentList | assignmentTuple) (syntaxComma (constant | assignmentMap | assignmentSet | assignmentList | assignmentTuple))*
    ;
```

Comment:
Please extract common parts into the separated rule:

```g4
expressionList
   : expression (syntaxComma expression)*
   ;

expression
   : constant
   | assignmentMap
   | assignmentSet
   | assignmentList
   | assignmentTuple
   ;
```

Reply:
Very neat.